ansible/playbooks/aix.yml

```diff
@@ -256,21 +264,32 @@
           register: java8
           tags: java8
 
-        - name: Create /usr/java8_64 if it does not exist
-          file:
-            path: /usr/java8_64
-            state: directory
-          when: java8.stat.isdir is not defined
-          tags: java8
-
         - name: Transfer and Extract AdoptOpenJDK 8
           unarchive:
             src: https://api.adoptopenjdk.net/v2/binary/releases/openjdk8?openjdk_impl={{ bootjdk }}&os=aix&arch=ppc64&release=latest&type=jdk
-            dest: /usr/java8_64
+            dest: /usr
             remote_src: yes
           when: java8.stat.isdir is not defined
           tags: java8
 
+        - name: Find java8 directory
+          find:
+            file_type: directory
+            paths: /usr
+            patterns: 'jdk8u*'
+          register: java8_files_matched
+          tags: java8
+
+        - name: Rename to java8_64
+          command: mv "{{ item.path }}" /usr/java8_64
```

Comment:
Can't we use the `file` module in ansible to create a symbolic link instead of moving in this situation now that we've resolved the wildcard problem? (Although I'm 50/50 on this one ... Maybe rename is ok)